modules/bitgo/src/v2/coins/utxo/recovery/crossChainRecovery.ts

```diff
@@ -47,8 +46,24 @@ class BitgoPublicApi {
    * @returns {*}
    */
   async getUnspentInfo(addresses: string[]): Promise<Unspent[]> {
-    const url = this.coin.url(`/public/addressUnspents/${_.uniq(addresses).join(',')}`);
-    return (await request.get(url)).body as Unspent[];
+    const res = await Bluebird.map(
```

Comment:
I just realized that at this point it would make sense to get rid of any duplicate addresses in `addresses`. I'm pushing a commit for it.